tinytag/tinytag.py

```diff
@@ -34,7 +34,8 @@
 
 import json
 import operator
-from collections import MutableMapping, OrderedDict, defaultdict
+from collections import OrderedDict, defaultdict
+from collections.abc import MutableMapping
```

Comment:
The project still seems to support Python 2.7, this will be an error in Python 2.7. Below might be a more compatible approach.

```python
try:
    from collections.abc import MutableMapping
except ImportError:
    from collections import MutableMapping
```

Reply:
Ah I see, yeah I didn't check it with python 2.7.
Your solution looks good and is recommended on [stackoverflow](https://stackoverflow.com/questions/53978542/how-to-use-collections-abc-from-both-python-3-8-and-python-2-7#comment99592161_53978543) as well.
I'll incorporate that.